imap_processing/swe/swe_cdf_attrs.py

```diff
@@ -0,0 +1,217 @@
+"""CDF attrs explanation.
```

Comment:
This is really nice documentation! I think it'd be nice to put all of this into the cdf_utils file? Maybe move that into a separate module even: `cdf/utils.py` for organization?

https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/dev/imap_processing/cdf_utils.py

Reply:
Maxine is grouping all cdf utils into a folder. I could move it there after her changes are in.